### ISSUE TITLE
A11y_trace.playwright.dev_Drop Playwright Trace to load_FocusOrder: Keyboard focus is moving to the background controls.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/playwright#29099]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS;#A11yTCS;#Playwright releases;#P3_trace.playwright.dev_Web_Jan2024;#A11ySev2;#ChromiumEdge;#DesktopWeb;#WCAG2.1.1;#Keyboard;#Win11;#P3_WCP;#A11yAuto;

### Environmental Details:
Application Name: trace.playwright.dev
#URL: https://trace.playwright.dev/

### OS & Browser Details:
Windows 11 Enterprise Insider Preview
OS build	26025.1000
Microsoft Edge
Version 122.0.2348.0 (Official build) dev (64-bit)
Chrome Version: Version 120.0.6099.225 (Official Build) (64-bit)

### Repro Steps:

1. Hit on this URL: https://trace.playwright.dev 
2. "Drop Playwright Trace to load" page will appear. 
4. Verify whether the keyboard focus is moving to the background controls or not.

### Actual Result:
Keyboard focus is moving to the background controls using Tab key.

### Expected Result:
Keyboard focus should not go to the background controls while navigating using Tab key.

### User Impact:
Keyboard users and screen reader users will get confused if keyboard focus going to background controls.

### Attachment:
![Keyboard](https://github.com/microsoft/playwright/assets/86241494/f376e315-978c-48dc-b5db-2dd3a69d9e3d)

Narrator -
![Narrator](https://github.com/microsoft/playwright/assets/86241494/e97ba132-49b6-46ca-bb59-b223ec397cb0)

**Comment by mxschmitt** [comment]
This one should be fixed on our "beta version" which is deployed here: https://trace.playwright.dev/next/

Next production release is in 4-6 weeks.

**Comment by Raghu-addepalli** [comment]
verified the issue in below environment issue is Fixed hence closing the bug and added attachment below,

https://trace.playwright.dev/next/


https://github.com/microsoft/playwright/assets/91586743/c6132f6b-ff26-493c-936e-9d7e963122dc

**Comment by SachinChavhan317** [comment]
#closed;
